robotlib/include/robotlib/proximity_handler.hpp

```diff
@@ -2,15 +2,18 @@
 
 #include <io-interface/ultra_sonic_sensor.hpp>
 
+using namespace iointerface::sensor;
+
 namespace robotlib::proximity_handler {
 
     class ProximityHandler {
     public: 
-        ProximityHandler(const double min_distance);
+        ProximityHandler(UltraSonicSensor us_sensor, const double min_distance);
```

Comment:
should be a vector no?

Reply:
should be a const reference as well?

Reply:
For the moment it isn’t a vector, as I wanted to start simple, with just one sensor. I’ll put it const.